app/src/controllers/organizations/projects/reducer.js

```diff
@@ -16,21 +16,20 @@
 
 import { combineReducers } from 'redux';
 import { fetchReducer } from 'controllers/fetch';
-import { paginationReducer } from 'controllers/pagination';
+import { alternativePaginationReducer } from 'controllers/pagination';
 import { loadingReducer } from 'controllers/loading';
 import { createPageScopedReducer } from 'common/utils/createPageScopedReducer';
-import { ORGANIZATION_PROJECTS_PAGE } from 'controllers/pages/constants';
-import { NAMESPACE, FETCH_ORGANIZATION_PROJECTS } from './constants';
+import { initialPaginationState, NAMESPACE } from './constants';
 
-export const projectFetchReducer = fetchReducer(FETCH_ORGANIZATION_PROJECTS, {
+export const projectFetchReducer = fetchReducer(NAMESPACE, {
   contentPath: 'items',
   initialState: [],
 });
 
 export const reducer = combineReducers({
-  pagination: paginationReducer(NAMESPACE),
+  pagination: alternativePaginationReducer(NAMESPACE, initialPaginationState),
   loading: loadingReducer(NAMESPACE),
   projects: projectFetchReducer,
 });
 
-export const projectsReducer = createPageScopedReducer(reducer, ORGANIZATION_PROJECTS_PAGE);
+export const projectsReducer = createPageScopedReducer(reducer, NAMESPACE);
```

Comment:
Please revert. As the `createPageScopedReducer` expects to receive a page, not namespace.

Reply:
Done.